### PR TITLE
update document

### PR DESCRIPTION
**Pull request by scarlett2018**

### CODE REVIEW

File: docs/WriteYourTrial.md [hunk 1]
```diff
@@ -12,32 +17,71 @@ We also support NNI APIs for trial code. By using this approach, you should firs
...
-```
-RECEIVED_PARAMS = nni.get_parameters()
-```
-to get hyper-parameters' values assigned by tuner. `RECEIVED_PARAMS` is an object, for example: 
-```
-{"conv_size": 2, "hidden_size": 124, "learning_rate": 0.0307, "dropout_rate": 0.2029}
-```
+>Step 2 - Update model codes
+~~~~
+2.1 Claim NNI API
```
Comment (xuehui1991) [inline docs/WriteYourTrial.md]: Claim? (I'm not sure....

File: docs/WriteYourTrial.md [hunk 2]
```diff
@@ -12,32 +17,71 @@ We also support NNI APIs for trial code. By using this approach, you should firs
     "learning_rate":{"_type":"uniform","_value":[0.0001, 0.1]}
 }
 ```
-You can refer to [here](SearchSpaceSpec.md) for the tutorial of search space.
+Refer to [Search Spece Spec](SearchSpaceSpec.md) to learn more about search space.
```
Comment (xuehui1991) [inline docs/WriteYourTrial.md]: I think Refer to [Search space] better.  Also here is a typo "Spece"

File: docs/WriteYourTrial.md [hunk 3]
```diff
@@ -1,9 +1,14 @@
-**Write a Trial which can Run on NNI**
+**Write a Trial Run on NNI**
 ===
-There would be only a few changes on your existing trial(model) code to make the code runnable on NNI. We provide two approaches for you to modify your code: `Python annotation` and `NNI APIs for trial`
 
-## NNI APIs 
-We also support NNI APIs for trial code. By using this approach, you should first prepare a search space file. An example is shown below: 
+A **Trial** in NNI is an individual attempts at applying a set of parameters on a model.
```
Comment (xuehui1991) [inline docs/WriteYourTrial.md]: individual?  or a experiment? (only individual concept in evolution???

File: docs/WriteYourTrial.md [hunk 4]
```diff
@@ -12,32 +17,71 @@ We also support NNI APIs for trial code. By using this approach, you should firs
...
-The assessor will decide which trial should early stop based on the history performance of trial(intermediate result of one trial).
-The tuner will generate next parameters/architecture based on the explore history(final result of all trials).
+        `nni.report_final_result(accuracy)` 
+        
+    to send `accuracy` to tuner. 
+~~~~
+
+**NOTE**: 
+~~~~
+accuracy - The `accuracy` could be any python data type, but  if you use NNI built-in tuner/assessor, `accuracy` should be a numerical variable (e.g. float, int).
```
Comment (xuehui1991) [inline docs/WriteYourTrial.md]: python object instead "python data type?"

File: docs/WriteYourTrial.md [hunk 5]
```diff
@@ -12,32 +17,71 @@ We also support NNI APIs for trial code. By using this approach, you should firs
...
+* annotate which variable you want to report as intermediate result to `assessor`
+* annotate which variable you want to report as the final result (e.g. model accuracy) to `tuner`. 
+
+Again, take MNIST as an example, it only requires 2 steps to write a trial with NNI Annotation.
+
+>Step 1 - Update codes with annotations 
+
+Please refer the following tensorflow code snippet for NNI Annotation, the highlighted 4 lines are annotations that help you to: (1) tune batch\_size and (2) dropout\_rate, (3) report test\_acc every 100 steps, and (4) at last report test\_acc as final result.
+
+>What noteworthy is: as these new added codes are annotations, it does not actually change your previous codes running dependencies or outputs, you can still run your code as usual in environments without NNI installed.
```
Comment (xuehui1991) [inline docs/WriteYourTrial.md]: change your previous codes running dependencies or outputs.   "change any your previous codes logic" better? (just a little suggestion...

File: examples/trials/mnist/config.yml [hunk 6]
```diff
@@ -2,7 +2,7 @@ authorName: default
 experimentName: example_mnist
 trialConcurrency: 1
 maxExecDuration: 1h
-maxTrialNum: 1
+maxTrialNum: 100
```
Comment (xuehui1991) [inline examples/trials/mnist/config.yml]: may have conflict with zejun's pr
